src/wallet/rpcdump.cpp

```diff
@@ -88,6 +88,38 @@ bool GetWalletAddressesForKey(CWallet * const pwallet, const CKeyID &keyid, std:
     return fLabelFound;
 }
 
+bool ImportPrivateKey(CWallet * const pwallet, const UniValue& privkey, const std::string& label = "", const int64_t timestamp = 1)
+{
+    CKey key = DecodeSecret(privkey.get_str());
+    if (!key.IsValid()) throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Invalid private key encoding");
+
+    CPubKey pubkey = key.GetPubKey();
+    assert(key.VerifyPubKey(pubkey));
+    CKeyID address = pubkey.GetID();
+    {
```

Comment:
In commit "[wallet] [rpc] Move private key import functionality out of importprivkey",

Remove this scope?